### DISCUSSION TITLE
1K

### DISCUSSION THREAD

**Discussion by vrodriguezf** [timeseriesAI/tsai#289] [General]
I just saw that tsai recently got 1K stars. Congratulations!!!! 

This thread is just to thank again Ignacio for all the effort and work that he is putting into this library, it is just mind blowing to me what this has become in the last year. This is now my one-stop shop for time series needs in all my projects, and a source of constant learning :)

**Comment by oguiza** [comment]
Wow! Thank you, @vrodriguezf!! 🙃
I really appreciate your kind words. This really motivates me to keep learning and sharing new models and methods in tsai 😃
I'm really overwhelmed by the support that you and many others have provided during this time.
